traffic_ops_ort/README.md

```diff
@@ -1,105 +1,3 @@
-
-# Special Processing
-
-Certain config files perform extra processing.
-
-## Global replacements
-
-All config files have certain text directives replaced. This is done by the atstccfg config generator before the file is returned to ORT.
-
-* `__SERVER_TCP_PORT__` is replaced with the Server's Port from Traffic Ops; unless the server's port is 80, 0, or null, in which case any occurrences preceded by a colon are removed.
-* `__CACHE_IPV4__` is replaced with the Server's IP address from Traffic Ops.
-* `__HOSTNAME__` is replaced with the Server's (short) HostName from Traffic Ops.
-* `__FULL_HOSTNAME__` is replaced with the Server's HostName, a dot, and the Server's DomainName from Traffic Ops (i.e. the Server's Fully Qualified Domain Name).
-* `__RETURN__` is replaced with a newline character, and any whitespace before or after it is removed.
-
-## remap.config
-
-ORT processes `##OVERRIDE##` directives in the remap.config file.
-
-The ##OVERRIDE## template string allows the Delivery Service Raw Remap Text field to override to fully override the Delivery Service’s line in the remap.config ATS configuration file, generated by Traffic Ops. The end result is the original, generated line commented out, prepended with ##OVERRIDDEN## and the ##OVERRIDE## rule is activated in its place. This behavior is used to incrementally deploy plugins used in this configuration file. Normally, this entails cloning the Delivery Service that will have the plugin, ensuring it is assigned to a subset of the cache servers that serve the Delivery Service content, then using this ##OVERRIDE## rule to create a remap.config rule that will use the plugin, overriding the normal rule. Simply grow the subset over time at the desired rate to slowly deploy the plugin. When it encompasses all cache servers that serve the original Delivery Service’s content, the “override Delivery Service” can be deleted and the original can use a non-##OVERRIDE## Raw Remap Text to add the plugin.
-
-## 50-ats.rules
-
-This is presumed to be a udev file for devices which are block devices to be used as disk storage by ATS.
-
-ORT verifies all devices in the file are owned by the owner listed in the file, and logs errors otherwise.
-
-ORT verifies all devices in the file do not have filesystems. If any device has a filesystem, ORT assumes it was a mistake to assign as an ATS storage device, and logs a fatal error.
-
-# Logging
-
-ORT outputs its immediate log to `stdout`, therefore it will log wherever you direct it. The recommended system location is `/var/log/ort/ort.log`.
-
-ORT uses the helper tool `atstccfg` to generate config files. Its log is output at `/var/log/ort/atstccfg.log`.
-
-# Trivia
-
-ORT stands for "Operational Readiness Test." The acronym is a legacy artifact and does not reflect the current purpose, which is to apply configuration from Traffic Ops.
+ORT, traffic_ops_ort.pl, is deprecated and will be removed in the next major version. Please use [/t3c/README.md](/uri "t3c").
```

Comment:
That link is broken: https://github.com/apache/trafficcontrol/blob/master/uri

I think for GitHub's renderer you can use relative path links like `./t3c` (but I didn't test that)

Reply:
Fixed